php/libraries/User.class.inc

```diff
@@ -284,20 +286,6 @@ class User extends UserPermissions implements \LORIS\StudyEntities\AccessibleRes
         return explode(';', $this->userInfo['Sites']);
     }
 
-     /**
-      * Get the user's site's ID number
-      *
-      * @return array
-      */
-    function getCenterID()
```

Comment:
It seemed silly to implement an interface just to throw an exception, so I removed this.